ga4gh/backend.py

```diff
@@ -782,23 +782,34 @@ def runSearchRequest(
         self.endProfile()
         return responseString
 
-    def runListReferenceBases(self, id_, requestArgs):
+    def runListReferenceBases(self, id_, requestJson):
         """
         Runs a listReferenceBases request for the specified ID and
         request arguments.
         """
-        compoundId = datamodel.ReferenceCompoundId.parse(id_)
+        # In the case when an empty post request is made to the endpoint
```

Comment:
This can't happen --- the body is required to be a valid JSON document by the spec.

In any case, we shouldn't be doing this here. You should be able to reuse the machinery used by all the other POST requests to take care of this. Start at the frontend and get rid of the special case for ListReferenceBases.

Should this still be called ListReferenceBases?